config/sidekiq.yml

```diff
@@ -3,3 +3,5 @@
 :queues:
   - default
   - mailers
+  - active_storage_analysis
+  - active_storage_purge
```

Comment:
Do we need this? I can't imagine we would need ActiveStorage in this application?

Reply:
I suppose we can avoid these two if we're sure we'll never use ActiveStorage. It was to be sure we can safely use all the new rails 6 defaults and specifically:
- `Rails.application.config.active_storage.queues.analysis = :active_storage_analysis`
- `Rails.application.config.active_storage.queues.purge = :active_storage_purge`

Reply:
I did a quick search and I don't see any mentions of it being used in the app, in fact it doesn't even seem to be `require`d at any point.

Reply:
Sounds good. I'll rebase and edit the commit to remove them.